### DISCUSSION TITLE
Suggestions for best practice to transfer consumed and processed files to another folder?

### DISCUSSION THREAD

**Discussion by BlankSlate56** [jonaswinkler/paperless-ng#850] [Support]
My initial Paperless setup was pointing the media output to the default docker/paperless/media folder, and I now have hundreds of documents at that location.  I want to change that target folder to a root-level folder that isn't buried so many subfolders deep.  I've read through the docs and have an idea as to how to point paperless to the new folder for any new documents that are generated.  However, I'm wondering what the best practice is to migrate the documents from the old default folder to the new folder without losing all of the database connections.  I'm guessing that I'll have to use the Document Exporter/Import management utilities, but wanted to check to see if there was a better way.

Thanks.

**Comment by jonaswinkler** [comment]
Hello!

* You don't have to use the exporter / impoerter.
* Paperless stores the location of files relative to the media folder. You can change the location of the media folder with configuration variables. With docker, you most likely want to change where the media folder is mounted on the host system.
* Shutting down paperless, moving the media folder, adjusting the docker configuration and starting the instance again should do the trick,
* There's also a sanity checker (https://paperless-ng.readthedocs.io/en/latest/administration.html#sanity-checker). I'd execute that before and after the operation. It will verify that all files exist at the configured location.

**Reply by BlankSlate56** [reply]
So the method that we used to point paperless to new consume and output locations was to remap the folders as shown below.  However, this still leaves the previously-processed docs in the original default location.

Should we have been using the PAPERLESS_MEDIA_ROOT  and PAPERLESS_CONSUMPTION_DIR variables instead?

![image](https://user-images.githubusercontent.com/78878227/113420892-bd45b280-938f-11eb-829d-2d5a997b299e.png)

**Reply by jonaswinkler** [reply]
When using docker, PAPERLESS_MEDIA_ROOT adjusts the location inside the container. It is "/src/src/paperless/media" by default. You don't need to change that. The files don't actually reside inside the container, your media folder at "./media" is mounted to "/usr/src/paperless/media" inside the container when the application starts. You really don't need to be concerned about where the files are located within the container.

If you want your files to be stored elsewhere on the host, simply change "./media" to a different folder, and move the files.

Does that answer your question?

**Reply by BlankSlate56** [reply]
So originally my folder structure was NAS/Volume2/docker/Paperless/media/documents/originals/Named Folders, where originals contained Named Folders that were created from my file naming rule (1 folder per correspondent).

Now, my folder structure is NAS/volume2/paperless-archive/Named Folders.  With this setup, I'm eliminating 4 subfolder levels, including the media folder.  I've tested it and paperless is outputing the files correctly into the paperless-archive directory I've created.  I personally really hate having stuff buried so many levels deep.  

With the setup outlined above, I'm not using a folder named media anymore.  Does that matter?  If I pull the files from NAS/Volume2/docker/Paperless/media/documents/originals/Named Folders and paste them in NAS/volume2/paperless-archive/Named Folders, will the database pick them up in the new location?
